app/assets/stylesheets/events.scss

```diff
@@ -0,0 +1,81 @@
+// Place all the styles related to the events controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+* {
+	margin: 0px;
+}
+
+.container-fluid {
+	padding: 0px;
+}
+
+#myCarousel ol li {
+	background-color: #169cb7;
+	border-color: white;
+	width: 15px;
+	height: 15px;
+	border-radius: 50%;
+	margin-right: 10px;
+}
+
+#carousel{
+ border: black solid thin;
+ padding: 0px;
+ margin-top: 20px;
+ // margin-left: 10px;
+}
+
+.img-responsive {
+	display: flex;
+	max-width: 100%;
+	height: 100vh;
+	// width: 1200px;
+	background-size: cover;
+	margin-right: 0px !important;
+}
+
+.join-btn-div {
+	position: absolute;
+	font-family: 'Open Sans', sans-serif;
+	top: 75%;
+	left: calc(50% - 101px);
+	z-index: 1;
+}
+
+.join-btn {
+	font-size: 26px;
+	color: white;
+	background-color:  #169cb7;
+}
+.join-btn:hover {
+	font-size: 26px;
+	color: #169cb7;
+	background-color:  yellow;
+}
+
+.details_and_map {
+	 margin-top: 15px;
+}
+
+.details {
+	color:  #169cb7;
+	font-family: 'Open Sans', sans-serif;
+	// text-align: center;
+	padding-left: 50px;
+}
+
+.details h1 {
+	padding-right: 40px;
+	font-weight: bold;
+	text-align: center;
+}
+.map-event {
```

Comment:
I would probably rename this to `event-map`, map-event sounds more like an event on the map, like a click or a drag (even though this is css). Assigning good variable/class names is one of the hardest things in programming, but when done right is immensely helpful for other people trying to understand your code.